### ISSUE TITLE
ImportError: No module named censys.certificates

### ISSUE THREAD

**Issue by remonsec** [censys/censys-python#37]
$ pip show censys
```
Name: censys
Version: 0.0.8
Summary: Python library for interacting with Censys Search Engine (censys.io)
Home-page: https://github.com/censys/censys-python
Author: Censys Team
Author-email: team@censys.io
License: Apache License, Version 2.0
Location: /usr/local/lib/python3.8/dist-packages
Requires: netaddr, requests
Required-by: 
```
$ python --version
```
Python 2.7.18rc1
```
pip install censys not working getting import error

**Comment by thehappydinoa** [comment]
Hi, I am unable to reproduce this on `macOS 10.15.6` running `Python 2.7.16`.

```python
>>> import censys.certificates
>>> censys.certificates.CensysCertificates(api_id="XXX", api_secret="XXX")
```

Would you be able to provide more context?

**Comment by thehappydinoa** [comment]
This appears to be a duplicate of #25

**Comment by tomsec** [comment]
I am also facing the same error:

Traceback (most recent call last):
  File "lib/censys/censys_subdomain_finder.py", line 3, in <module>
    import censys.certificates
ImportError: No module named censys.certificates

Any clue on how to resolve this?

**Comment by thehappydinoa** [comment]
> I am also facing the same error:
> 
> 
> 
> Traceback (most recent call last):
> 
>   File "lib/censys/censys_subdomain_finder.py", line 3, in <module>
> 
>     import censys.certificates
> 
> ImportError: No module named censys.certificates
> 
> 
> 
> Any clue on how to resolve this?

This seems as if you are using a third party implementation of the Censys library. However, we have an update in the works that may be able to clear up some of the issues you are having.

**Comment by thehappydinoa** [comment]
Hi,

I believe this issue has been solved so I am closing it. If you have more questions feel free to reach out.

**Comment by BeeruxBash** [comment]
Hey!
I am facing the same issue even after trying the solutions mentioned in this thread as well as in #25. I am running the Censys analyzer in Cortex and I have provided the Censys Secret & API key parameter as uid and api key in the analyzer config. But whenevr I try to run it I face the following error:

Traceback (most recent call last):
  File "censys_analyzer.py", line 3, in <module>
    from censys.certificates import CensysCertificates
ModuleNotFoundError: No module named 'censys.certificates'

I've authenticated the credentials using the censys config but to no avail. Can someone please suggest me a solution.
TIA

**Comment by thehappydinoa** [comment]
Hi @BeeruxBash,

It appears that Cortex has yet to upgrade to use the latest `censys` package. If you update the import to `from censys.search import CensysCertificates` it should work.

**Comment by BeeruxBash** [comment]
Update
I changed the import to from censys.search import CensysCertificates and the error went away but unfortunately another error emerged staing:

Traceback (most recent call last):
  File "censys_analyzer.py", line 4, in <module>
    from censys.ipv4 import CensysIPv4
ModuleNotFoundError: No module named 'censys.ipv4'

I've searched this error online too but got no answer to resolve this issue.
Also, I was wondering that if these issues are related to the parameters I am giving to the Censys.io analyzer in Cortex as it asks for uid for which I have provided the Secret value given on the Censys website and API key for which I have provided the API key value given on the Censys website too. So, am I giving the correct values or not that maybe causing errors?
